resources/validations/src/ssp.sch

```diff
@@ -95,12 +98,12 @@
             github.com/usnistgov/oscal-content are not. The 0f78f05 commit is the
             most recent triggered by the OSCAL 1.0.0-rc1 release. Change this url
             accordingly if you know what you are doing.
-        -->
-            <profile href="https://raw.githubusercontent.com/usnistgov/oscal-content/0f78f05f0953e64f37b5cb24e0522030d82fc1fa/fedramp.gov/xml/FedRAMP_LOW-baseline-resolved-profile_catalog.xml"
+            -->
+            <profile href="{concat($baselines-base-path, '/FedRAMP_rev4_LOW-baseline_profile.xml')}"
```

Comment:
B: So I will comment on this and maybe the others, but you will want the resolved catalog baselines, as the profiles just explain how to compose some or "all" controls with a special shorthand, so a lot of the data in the resolved catalog baselines is more useful for our purposes, or you would essentially need to engage in profile resolution yourself.

If you try this, I bet you get a lot more or a lot less failures than normal with our test SSP as a bunch of required data we load alongside the FedRAMP values are missing.

This is why I recommend not pulling from `/src/` sub-directory, as these are generated by the CI-CD process in GSA/fedramp-automation and usnistgov/oscal-content if you look, and this will cause you a lot of headache for little gain for the actual validation development. (Read the profile resolution spec, it will be multiple epics of work.)


```suggestion
            <profile href="{concat($baselines-base-path, '/FedRAMP_rev4_LOW-baseline-resolved-profile_catalog.xml')}"
```